build_tools/cmake/build_host_and_riscv.sh

```diff
@@ -0,0 +1,94 @@
+#!/bin/bash
```

Comment:
Realized this file is renamed from `build_riscv.sh`. Feel free to ignore those comments.

Reply:
Yeah I want to just delete this file shortly :-)